server/data/risksAndNeeds/risksAndNeedsApi.ts

```diff
@@ -70,6 +70,11 @@ function risksAndNeedsApiGetBuilder(username) {
       if (error.response?.status === 404) {
         return {}
       }
+      // 403 errors are returned for crns of restricted access people, we will record how many these occur for but continue as if they do not have a rosh score
+      if (error.response?.status === 403) {
+        logger.info('Risks and needs API: restricted crn requested')
```

Comment:
worth logging the crn here, or do you already have it?

Reply:
It's in the request since it's a get request